packages/front-end/tsconfig.json

```diff
@@ -4,15 +4,6 @@
         "baseUrl": "./",
         "jsx": "react"
     },
-    "paths": {
```

Comment:
Again, i think we need this to avoid the typescript error.

Reply:
It should work. Please check the latest changes. Lerna handles it.